### ISSUE TITLE
Separate feature generation code

### ISSUE THREAD

**Issue by redshiftzero** [dssg/police-eis#8]
Right now, most feature generation is done on the fly as models are trained. We should separate this feature generation code so that it runs when we receive new data or define new features. This should be based on what is currently in `eis/features/`. The feature generation code should for all examples and for a given `fake_today` generate the features and store them back in the database. The main modeling code should then just 1) select the relevant rows and 2) select the relevant features (defined in the config).

**Comment by hinnefe2** [comment]
I think it should be possible to keep using the existing FeatureLoader class (in eis/dataset.py) to load the features once we switch to storing the features in the database. Right now it seems like it reads the sql query stored in eis/features/officers.py for whatever feature it's loading, so we could presumably just change the sql to query to read from our features table instead. Hopefully this would save a lot of work adapting the rest of the pipeline to use stored features.

**Comment by redshiftzero** [comment]
Good point @hinnefe2! The advantage of this approach is that it will preserve most of the code, though it will mean that we are making a lot of queries and will still be joining in pandas. We could make a further change to `FeatureLoader` to select in SQL only the relevant columns from the database (thus no pandas). We could use the `name_of_features` attribute in the class to reference to the columns in the database that feature class corresponds to, and then have the FeatureLoader class collect those features. At that point then `FeatureLoader` can make a single SQL query to select only the relevant columns.
